tests/common.go

```diff
@@ -4397,6 +4399,79 @@ func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
 	}
 }
 
+func collectAndCopyStorkLogsForPxBackupTest(testCaseName string) {
+	testCaseName = strings.ReplaceAll(testCaseName, " ", "")
+	storkLabel := make(map[string]string)
+	storkLabel["name"] = "stork"
+	pxNamespace, err := Inst().V.GetVolumeDriverNamespace()
+	if err != nil {
+		log.Errorf("Error in getting stork namespace, Err: %v", err.Error())
+		return
+	}
+	podList, err := core.Instance().GetPods(pxNamespace, storkLabel)
+	if err != nil {
+		log.Errorf("Error in getting stork pods, Err: %v", err.Error())
+		return
+	}
+	masterNode := node.GetMasterNodes()[0]
+	err = runCmd("pwd", masterNode)
+	if err != nil {
+		log.Errorf("Error in running [pwd] command in node [%s]", masterNode.Name)
+		return
+	}
+	testCaselogDirPath := fmt.Sprintf("%s/%s-logs", pxbLogDirPath, testCaseName)
+	log.Infof("Creating a directory [%s] in node [%s] to store logs for the test case [%s]", testCaselogDirPath, masterNode.Name, testCaseName)
+	runCmd(fmt.Sprintf("mkdir -p %v", testCaselogDirPath), masterNode)
```

Comment:
Good catch! I will handle the error for mkdir -p command. In case of a failure of the runCmd function, it returns an error message that includes the command it ran. I am assuming, pwd might be more useful for de-bugging.

Reply:
cool! that'll work.